syntax_and_semantics/compile_time_flags.md

```diff
@@ -9,9 +9,15 @@ x86_64-unknown-linux-gnu
 # so the flags are: x86_64, unknown, linux, gnu
 ```
 
+To define a flag, simply use the `--define` or `-D` option, like so:
+
+```bash
+$ crystal some_program.cr -D my_flag
```

Comment:
Maybe add `crystal some_program.cr --debug my_flag` and  `crystal some_program.cr -Dmy_flag` as alternatives?

Reply:
Can you clarify what I need to change? `-D` to `-Dmy_flag` or `--define my_flag`? I'd prefer using `--define my_flag` since it distinguishes the option from the value, but `-Dx86_64` could also be an option (feels  better than `my_flag` too).

Also when you said `--debug` did you mean `--define`?

Reply:
Oh, it was late... of course I meant `--define`.
You don't *need* to change anything. I would suggest to show the usage at least of using `-D` without whitespace. But I don't think `-Dx86_64` is a good idea because this flag is actually set automatically based on the target triple.

Reply:
Personally I use `-Dflag`.